browser-extension/addTournesolRecommendations.css

```diff
@@ -28,25 +28,64 @@
 
 .language_option {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   float: right;
   margin-top: 3px;
-  margin-left: 4px;
+  margin-left: 2px;
 }
 
 #tournesol_link {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   position: absolute;
   right: 0px;
   top: 15px;
 }
 
+#refresh_button {
+  float: right;
+  margin-top: 0px;
+  margin-left: 12px;
+  margin-right: 12px;
+}
+
 #tournesol_icon {
   position: absolute;
   left: 0px;
 }
 
+.icon {
+  -webkit-filter: opacity(50%);
+  filter: opacity(50%);
+  width: 24;
+  height: 24;
+  border-radius: 12px;
+  color: #f8f5e8;
+}
+
+.icon:hover {
+  -webkit-filter: opacity(100%);
+  filter: opacity(100%);
+
+  -webkit-transition: all 0.1s ease;
+  -moz-transition: all 0.1s ease;
+  -o-transition: all 0.1s ease;
+  -ms-transition: all 0.1s ease;
+  transition: all 0.1s ease;
+}
+
+.icon:active {
+  -webkit-filter: opacity(100%);
+  filter: opacity(100%);
+  background: #bdbcb2;
+
+  -webkit-transition: all 0.1s ease;
+  -moz-transition: all 0.1s ease;
+  -o-transition: all 0.1s ease;
+  -ms-transition: all 0.1s ease;
+  transition: all 0.1s ease;
+}
```

Comment:
The button style is barely visible on youtube dark mode 

<img width="111" alt="Screenshot 2022-01-06 at 21 48 40" src="https://user-images.githubusercontent.com/8818081/148451579-9392a62c-73bc-4fe0-af63-18cc4d0bcc6d.png">

I believe you may use something like https://fontawesome.com/v5.15/icons/sync?style=solid for a similar icon and it would be modified by the css attribute `color: var(--yt-spec-text-primary);` like the title and text, unlike the png file that you currently use.

Reply:
I need help to do this. Why this js extension it isn't as simple as doing "npm install" and my attempts failed...

Reply:
No dependency is necessary. Instead of a .png image, we need to use a .svg element where CSS rules can be applied.
There is a catch though: `<img src="image.svg">` cannot by styled using CSS. We need to inject the `<svg>` element in the DOM.  

Here is a solution:
```js
refresh_button = document.createElement('button');
refresh_button.setAttribute('id', 'refresh_button');
fetch(chrome.runtime.getURL('images/refresh.svg'))
  .then(r => r.text())
  .then(svg => refresh_button.innerHTML = svg);
```

As the `<svg>` is now present inside the `<button>`, the CSS variable provided by youtube.com can be used to change the color:
```css
#refresh_button {
  fill: var(--yt-spec-text-primary);
}
```
Note that `fill` is used instead of `color` to color SVG elements.